### PR TITLE
[usecase-cordova] Add AdMob tests for Cordova 3.6/4.0

### PR DESCRIPTION
**Pull request by haoxli**
Impacted tests(approved): new 1, update 0, delete 0
Unit test platform: [Memo Pad][Crosswalk 15.43.347.0]
Unit test result summary: pass 1, fail 0, block 0

BUG=https://crosswalk-project.org/jira/browse/XWALK-2429

### CODE REVIEW

File: usecase/usecase-cordova-android-tests/samples/AdMob/REAMDE.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## Usecase Design
+
+Feature comes from https://crosswalk-project.org/jira/browse/XWALK-3097
+This sample demonstrates app works fine when two pages load each other for more than 512 times.
```
Comment (Honry) [inline usecase/usecase-cordova-android-tests/samples/AdMob/REAMDE.md]: Do you copy this file from other test without modification?

**Comment by Honry** [comment]
I know you have a precondition to install Admob plugin while building, please send this note to stakeholders and update relative document.

File: usecase/usecase-cordova-android-tests/samples/AdMob/REAMDE.md (hunk 1 above)
Comment (haoxli) [inline usecase/usecase-cordova-android-tests/samples/AdMob/REAMDE.md]: Copy a wrong README file.

**Comment by haoxli** [comment]
Fail to build admob plugin in Cordova 4.0, track this issue by https://github.com/floatinghotpot/cordova-plugin-admob/issues/186.

**Comment by Honry** [comment]
@haoxli, any progress? About the different result on Cordova 3.6 and 4.0, I guess if it is caused by different version of admob plugin required by different version of Cordova?

**Comment by haoxli** [comment]
@Honry  Yes, we need use different version of admob pluginf for Cordova 3.6 and 4.0 but the test fail to run with the plugin for cordova 4.0, I haven't got response from floatinghotpot.

File: usecase/usecase-cordova-android-tests/README.md [hunk 2]
```diff
@@ -11,6 +11,18 @@ It has these features:
 
 * Lin, Wanming <wanmingx.lin@intel.com>
 
+## Pre-condition
+
+There are third-part plugins needed for cordova usecase:
+AdMob3.6: https://github.com/floatinghotpot/cordova-plugin-admob
+AdMob4.0: https://github.com/floatinghotpot/cordova-admob-pro
+
+Make a directory named "thirdpart\_plugins", and git clone these plugins in it.
```
Comment (Honry) [inline usecase/usecase-cordova-android-tests/README.md]: thirdparty_plugins, why add the '\'?

Comment (haoxli) [inline usecase/usecase-cordova-android-tests/README.md]: It's a escape character

File: usecase/usecase-cordova-android-tests/README.md [hunk 3]
```diff
@@ -11,6 +11,18 @@ It has these features:
 
 * Lin, Wanming <wanmingx.lin@intel.com>
 
+## Pre-condition
+
+There are third-part plugins needed for cordova usecase:
```
Comment (Honry) [inline usecase/usecase-cordova-android-tests/README.md]: third-party

**Comment by Honry** [comment]
LGTM

**Pull request merged**